onset_maproom/calc.py

```diff
@@ -293,6 +308,59 @@ def seasonal_onset_date(
     # + seasonal_onset_date.onset_delta
     return seasonal_onset_date
 
+def seasonal_cess_date(
+    search_start_day,
+    search_start_month,
+    search_days,
+    soil_moisture,
+    dry_thresh,
+    min_dry_days,
+    time_coord="T"
+):
+    # Deal with leap year cases
+    if search_start_day == 29 and search_start_month == 2:
+        search_start_day = 1
+        search_start_month = 3
+
+    # Find an acceptable end_day/_month
+    first_end_date = soil_moisture[time_coord].where(
+        lambda x: (x.dt.day == search_start_day) & (x.dt.month == search_start_month),
+        drop=True,
+    )[0] + np.timedelta64(
+        search_days,
+        "D",
+    )
+
+    end_day = first_end_date.dt.day.values
+
+    end_month = first_end_date.dt.month.values
+
+    # Apply daily grouping by season
+    grouped_daily_data = daily_tobegroupedby_season(
+        soil_moisture, search_start_day, search_start_month, end_day, end_month
+    )
+    # Apply onset_date
+
+    seasonal_data = (
+        grouped_daily_data["soil_moisture"]
```

Comment:
use seasonal_moisture.name maybe (see seasonal_onset...)? because you can't be sure that the name of the input argument will be "soil_moisture," correct?